python/tests/parsers/test_crawler_current_events.py

```diff
@@ -1,44 +1,63 @@
 # pylint: disable=redefined-outer-name
 """This file contains tests for crawler_current_events"""
+from datetime import date
+
 import pytest
 
 from edgar_filing_searcher.parsers.crawler_current_events import ensure_13f_filing_detail_urls, \
-    parse_13f_filing_detail_urls
-from edgar_filing_searcher.errors import NoUrlException
+    get_subdirectories_for_specific_date, generate_dates
+
+DATE_1 = date(2021, 1, 8)
+DATE_2 = date(2021, 1, 9)
+DATE_3 = date(2021, 1, 10)
 
 
 @pytest.fixture
-def edgar_current_events_text():
+def subdirectories():
```

Comment:
This doesn't need to be a fixture. Just make it a variable like the dates above.